build/sync/plan.yml

```diff
@@ -0,0 +1,49 @@
+checks:
+  - type: repo_is_clean
+    params:
+      repo: template
+  - type: repo_is_clean
+    params:
+      repo: plugin
+actions:
+  - paths:
+      - build
+    actions:
+      - type: overwrite_directory
+        params:
+          create: true
+  - paths:
+      - Makefile
+      - .editorconfig
+    actions:
+      - type: overwrite_file
+        params:
+          create: true
+        conditions:
+          - type: file_unaltered
+            params:
+              repo: plugin
+              reference-repo: template
```

Comment:
I don't think `file_unaltered` should be checked for `Makefile`. The `Makefile` from this repo should be the golden standard and any modifications should go into `build/custom.mk`.

Reply:
Good suggestion. Thank you.